extensions/amp-ad-network-adzerk-impl/0.1/data/0.template

```diff
@@ -1,24 +1,19 @@
-<!doctype html><html amp4ads><head><meta charset="utf-8"><script async src="https://cdn.ampproject.org/amp4ads-v0.js"></script>
-    <style amp4ads-boilerplate>body{visibility:hidden}</style>
+<!doctype html>
+<html ⚡>
+  <head>
+    <meta charset="utf-8">
+    <link rel="canonical" href="self.html" />
     <meta name="viewport" content="width=device-width,minimum-scale=1">
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.1.js"></script>
   </head>
-  <body><template type="amp-mustache">
-    <a href={{url}} rel="nofollow" target="_blank">
-      <amp-img src={{externalUrl}} width="250px" height="150px" layout="responsive"></amp-img>
-    </a>
-    <script amp-ad-metadata type=application/json>
-    {
-      "ampRuntimeUtf16CharOffsets" : [ 57, 127 ],
-      "customElementExtensions": [
-        "amp-bind"
-      ],
-      "extensions": [
-        {
-          "custom-element": "amp-bind",
-          "src": "https://cdn.ampproject.org/v0/amp-bind-0.1.js"
-        }
-      ]
-    }
-    </script>
-    </template></body>
+  <body>
+    <template type="amp-mustache">
+      <a href="{{url}}" rel="nofollow" target="_blank" title="{{title}}">
```

Comment:
`url` => `clickUrl`

Reply:
"url" and "externalUrl" are both values given by AdZerk in their original template. Should I still change?